.circleci/config.yml

```diff
@@ -149,8 +149,8 @@ commands:
           name: Build py27py36 Wagon
           command: |
               git clone https://github.com/cloudify-cosmo/cloudify-wagon-build-containers.git
-              docker build -t cloudify-centos-7-wagon-builder cloudify-wagon-build-containers/centos_7_py2py3
-              docker run -v ~/project/:/packaging cloudify-centos-7-wagon-builder
+              docker build -t cloudify-wagon-builder-centos-7-py2py3 cloudify-wagon-build-containers/centos_7_py2py3
```

Comment:
Why you change the tag name?

Reply:
My suspicion is that circle ci is using a cached image and that's why it's failing. Also this name is better.

Reply:
OK!